tools/create_audio_messages.py

```diff
@@ -18,7 +18,6 @@
     argparser.add_argument('-o', '--output', type=str, default='sd-card', help='The directory where to create the audio messages. (default: `sd-card`)')
     text_to_speech.addArgumentsToArgparser(argparser)
     argparser.add_argument('--skip-numbers', action='store_true', help='If set, no number messages will be generated (`0001.mp3` - `0255.mp3`)')
-    argparser.add_argument('--only-new', action='store_true', help='If set, only new messages will be created.')
```

Comment:
I'm rather new to TonUINO, but I've used the "--only-new" argument a couple of times already. I would find it unfortunate if this would be removed.

Reply:
We can add this back, or merge this option in. I don't mind, the goal is to update the "base version" to what is here https://github.com/seisfeld/TonUINO/tree/master/tools . All future changes should be on top of that. This can't be rocket science. ;)